src/views/blocks/rsvp/form.php

```diff
@@ -8,30 +8,34 @@
  *
  * See more documentation about our Blocks Editor templating system.
  *
- * @link {INSERT_ARTICLE_LINK_HERE}
+ * @link    {INSERT_ARTICLE_LINK_HERE}
  *
- * @since 4.9
- * @since 4.12.0 Add $post_id to filter for hiding opt-outs.
+ * @since   4.9
+ * @since   4.12.0 Add $post_id to filter for hiding opt-outs.
+ * @since   TBD Add vars to docblock.
  *
  * @version 4.12.0
  *
+ * @var Tribe__Tickets__Editor__Template $template
+ * @var Tribe__Tickets__RSVP             $rsvp
+ * @var int                              $post_id
  */
 
 $going      = $this->get( 'going' );
-$must_login = ! is_user_logged_in() && tribe( 'tickets.rsvp' )->login_required();
+$must_login = ! is_user_logged_in() && $rsvp->login_required();
 ?>
 <!-- This div is where the AJAX returns the form -->
 <div class="tribe-block__rsvp__form">
 	<?php if ( ! empty( $going ) && ! $must_login ) :
 		$ticket = $this->get( 'ticket' );
-		$args = [
+		$args   = [
 			'ticket_id' => $ticket->ID,
 			'post_id'   => $post_id,
-			'ticket'    => tribe( 'tickets.rsvp' )->get_ticket( get_the_id(), $ticket->ID ),
+			'ticket'    => $rsvp->get_ticket( get_the_id(), $ticket->ID ),
```

Comment:
Is `$ticket` not an RSVP object already? Also if we are using `$this` for the template below it wouldn't need to pass down again. We can likely remove all of the args except the `ticket_id` one which might be needed separately by the template. It might already be passed into this current template though, so you might want to check that too.

Reply:
yes, `$ticket` - good catch and ticket_id was able to be replaced with $ticket->ID